src/span.cpp

```diff
@@ -38,7 +38,7 @@ uint64_t SpanData::spanId() const { return span_id; }
 
 std::unique_ptr<SpanData> makeSpanData(std::string type, std::string service,
                                        ot::string_view resource, std::string name,
-                                       uint64_t trace_id, int64_t span_id, uint64_t parent_id,
+                                       uint64_t trace_id, uint64_t span_id, uint64_t parent_id,
```

Comment:
Oh shiiiiiiii

I think I missed a -Wall somewhere.

Reply:
Even `-Wall -Wextra -pedantic` wouldn't have made noise about it.